openquake/qa_tests_data/classical_tiling/case_2/models/site/site_model.xml

```diff
@@ -8,7 +8,7 @@ xmlns:gml="http://www.opengis.net/gml"
         <site lat="37.54648" lon="34.98430" vs30="760.00" vs30Type="inferred" z1pt0="100" z2pt5="5"/>
         <site lat="36.91276" lon="35.17898" vs30="203.00" vs30Type="inferred" z1pt0="100" z2pt5="5"/>
         <site lat="36.97022" lon="35.42197" vs30="289.12" vs30Type="inferred" z1pt0="100" z2pt5="5"/>
-        <site lat="37.56571" lon="38.49118" vs30="457.76" vs30Type="inferred" z1pt0="100" z2pt5="5"/>
+        <site lat="37.56571" lon="38.49118" depth="-1.0000" vs30="457.76" vs30Type="inferred" z1pt0="100" z2pt5="5"/>
```

Comment:
Actually we forgot to change yet another place where the mesh was build, but now I fixed it, see commit https://github.com/gem/oq-engine/pull/2509/commits/c05fe19dfaa5a0d9368b445cc2b860aa97fd239c